source/convection_diffusion/assemble_constant_matrices.cc

```diff
@@ -19,11 +19,8 @@ void HeatEquation<dim>::assemble_constant_matrices()
   mass_matrix       = 0.;
   stiffness_matrix  = 0.;
 
-  // Compute the highest polynomial degree from all the integrands
-  const int p_degree = 2 * temperature->fe_degree;
-
   // Initiate the quadrature formula for exact numerical integration
-  const QGauss<dim>   quadrature_formula(std::ceil(0.5 * double(p_degree + 1)));
+  const QGauss<dim>   quadrature_formula(temperature->fe_degree);
```

Comment:
This is wrong

Reply:
How is this wrong? In this method the mass matrix and the stiffness matrix are assembled simultaneously. As such, the highest polynomial is two times the polynomial degree of the Lagrange element. The `std::ceil` call here is based in the formula to assured that the quadrature rule used in the numerical integration produces the same result as its analytical integration.

Reply:
These comments were just a note for me that
```c++
   const QGauss<dim>   quadrature_formula(temperature->fe_degree); 
```
is wrong. The comment is outdated.